flows/actions/transfer_airtime.go

```diff
@@ -65,17 +64,9 @@ func (a *TransferAirtimeAction) transfer(run flows.Run, logEvent flows.EventCall
 	// fail if we don't have a contact
 	contact := run.Contact()
 
-	// fail if the contact doesn't have a tel URN
-	telURNs := contact.URNs().WithScheme(urns.Phone.Prefix)
```

Comment:
What about updating `WithScheme` to take multiple schemes..

```go
// WithScheme returns a new URN list containing of only URNs of the given schemes
func (l URNList) WithScheme(schemes ...string) URNList {
	var matching URNList
	for _, u := range l {
		if slices.Contains(schemes, u.urn.Scheme()) {
			matching = append(matching, u)
		}
	}
	return matching
}
```

Am not loving `Contact.GetAirtimeSenderRecipient()` especially since this feature might be removed.. don't want it polluting too many files outside of this action.